tools/pythonpkg/tests/fast/pandas/test_pandas_df_none.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+import pytest
+import duckdb
+import sys
+import gc
+
+
+class TestPandasDFNone(object):
+    def test_none_deref(self):
+        con = duckdb.connect()
+        gc.collect()
+        none_refcount = sys.getrefcount(None)
+        for _ in range(1000):
+            df = con.sql("select NULL::VARCHAR as a").df()
+
+        gc.collect()
+        after = sys.getrefcount(None)
+        # Verify that we don't deref None when creating a Pandas DataFrame
+        assert after >= none_refcount
```

Comment:
I don't think the refcount of None is something we can reliably assert on (especially since in 3.12+ it's effectively imortal, see https://github.com/python/cpython/pull/19474) - just having the code that can cause the issue is probably enough

Reply:
@Tishj could we still change the test? Otherwise LGTM

Reply:
So we just want to remove the test then?
I figured if it's going to be immortal that means `after >= none_refcount` should still be true

Reply:
Just leave the test as-is but remove the checks on `sys.getrefcount` perhaps?

Reply:
I'm guessing the actual number is dependent on what has happened before perhaps?

Reply:
I would say just remove the checks on refcount to make sure this test doesn't turn into a sporadically failing test in the future, and leave it as-is

Reply:
Sure, I just think the test is not verifying anything anymore so it might make more sense to just delete it altogether